src/components/timeline/TrackContextMenu.js

```diff
@@ -516,13 +517,46 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     );
   }
 
+  renderHideAllTracksByType() {
+    const { rightClickedTrack, globalTracks, localTracksByPid } = this.props;
+    if (rightClickedTrack === null) {
+      return null;
+    }
+    const trackIndex = rightClickedTrack.trackIndex;
+    if (rightClickedTrack.type === 'global') {
+      return (
+        <MenuItem
+          key={trackIndex + 'hide-all'}
+          preventClose={false}
+          data={(rightClickedTrack, globalTracks, localTracksByPid)}
+          onClick={this._toggleGlobalTrackVisibility}
+        >
+          Hide all {`"${this.getRightClickedTrackName(rightClickedTrack)}"`}{' '}
+          tracks
+        </MenuItem>
+      );
+    }
+    return (
+      <MenuItem
+        key={trackIndex}
+        preventClose={false}
+        data={(rightClickedTrack, globalTracks, localTracksByPid)}
+        onClick={this._toggleLocalTrackVisibility}
```

Comment:
So it's correct that now we have an item that says `Hide all "xyz" tracks` in the context menu. But this onClick event is still using the function for `Hide "xyz"` item in the context menu. We need to create our own function for this so we can hide the current track and all the tracks that have the same type.